2018/2018-11-19.md

```diff
@@ -58,7 +58,7 @@
 
 - Tudors: 
     - Make them collapsible
-    - ~Roughly �15-20 worth of parts
+    - Roughly 15 - 20 worth of parts
```

Comment:
I'd hardly call a pound sign "horrible", and removing it is impacting the clarity of the text.

If there's some problem with something not accepting non-ASCII characters, we should fix it rather than working around it.

Reply:
It wasn't a normal pound sign. These two files specifically had encoding that was breaking `mkdocs`.